src/SIL.XForge.Scripture/ClientApp/src/app/shared/share/share-dialog.component.html

```diff
@@ -1,12 +1,90 @@
 <ng-container *transloco="let t; read: 'share_dialog'">
-  <h1 mat-dialog-title>{{ t("share_with_others") }}</h1>
-  <app-share-control
-    mat-dialog-content
-    class="share-dialog"
-    [projectId]="data.projectId"
-    [defaultRole]="data.defaultRole"
-  ></app-share-control>
-  <div mat-dialog-actions align="end">
-    <button id="close-btn" mat-flat-button mat-dialog-close>{{ t("done") }}</button>
+  <h1 mat-dialog-title>
+    <span>{{ t("share") }} '{{ projectName }}'</span
```

Comment:
In general when we use a project name in a title we don't quote it, which would be better for consistency.

Also, not every language puts the verb before the object. This should be something like `t('share_project', {project: projectName})`